Procfile

```diff
@@ -0,0 +1,2 @@
+passenger: passenger start
+puma: puma -C config/puma.rb
```

Comment:
@ViditChitkara Is this the proper puma command or are there any other flags needed?